frontend/app/routes/protected/multi-channel/finalize-request.tsx

```diff
@@ -127,7 +128,21 @@ export default function PidVerification({ loaderData, actionData, params }: Rout
             />
             <h2 className="font-lato text-2xl font-semibold">{t('protected:finalize-request.final-check')}</h2>
             <p>{t('protected:finalize-request.first-time')}</p>
-            <p>{t('protected:finalize-request.before-processing')}</p>
+            <p>
+              <Trans
+                i18nKey="protected:finalize-request.before-processing"
+                components={{
+                  sarpLink: (
+                    <InlineLink
+                      to={t('protected:finalize-request.sarp-link')}
+                      className="external-link"
+                      newTabIndicator
+                      target="_blank"
```

Comment:
Does it make sense to combine these into a unified `newTab: boolean` property that will cause the link component to add `target="_blank"` and the indicator?

Or, to rephrase: would you ever specify a `newTabIndicator` prop *without* `target=_blank`?